test/test_response.py

```diff
@@ -745,6 +758,22 @@ def test_read_not_chunked_response_as_chunks(self):
         with pytest.raises(ResponseNotChunked):
             next(r)
 
+    def test_read_chunked_not_supported(self):
```

Comment:
We might be able to implement this one without mocks, try passing an `io.BytesIO` to `HTTPResponse` and reading from it with `read_chunked()`? Seems to be what is called out in the PR adding this functionality: https://github.com/urllib3/urllib3/pull/990

Reply:
Thanks, done.